misc/formatter/modules/formatter-core/src/test/resources/actions/query/assert/query_action_4.bal

```diff
@@ -0,0 +1,5 @@
+public function main() {
+    var c = from var i in 0 ... 5
+        select i
+        where i %2 != 0;
```

Comment:
```suggestion
        where i % 2 != 0;
```
Is it possible to make this space?

Reply:
Are you suggesting that the formatter should add the space after `%` in this case.

Reply:
Since this is a Invalid Node and this is a syntax error the formatter is not aware of the context. Thus if the user have not specified a whitespace this does not add a whitespace

Reply:
Okay. I didn't see there is an syntax error there.

Reply:
Can you point tests where this formatting is working?

Reply:
I updated the approach to describe what this PR does and the test added to the PR and the tests skipped [here](https://github.com/ballerina-platform/ballerina-lang/blob/master/misc/formatter/modules/formatter-core/src/test/java/org/ballerinalang/formatter/core/ParserTestFormatter.java#L93C18-L95) are the tests this PR fixes